src/os/bluestore/KernelDevice.cc

```diff
@@ -46,6 +46,7 @@ KernelDevice::KernelDevice(aio_callback_t cb, void *cbpriv)
 {
   zeros = buffer::create_page_aligned(1048576);
   zeros.zero();
+  rotational = true;
```

Comment:
Sure, I thought since this change is already done, these two should merge somewhere later. I have picked up function names from https://github.com/ceph/ceph/pull/9137 so that merge time they complement each other.  do you want me to pick changes from there ?